### DISCUSSION TITLE
Enable Aero Snap or WinTile in LXQt for Lubuntu

### DISCUSSION THREAD

**Discussion by ruppert-frederick** [lxqt/lxqt#2277] [Q&A]
Is there a way to enable the window snap by dragging it and bouncing it to the left of right like in Windows OS to have window resize half the screen ? I am missing this feature in the latest Lubuntu with LXQt.
Thank you,
Ruppert

**Comment by tsujan** [comment]
It's the job of Window Manager. Most WMs can do it; KWin can, for example. I remember that Openbox could do it too, but it's a long time I haven't used it.
